Docs build is failing

## 🐛 Bug

We're getting a 

```
Extension error:
Could not import extension sphinx_tags (exception: cannot import name 'get_matching_files' from 'sphinx.util.matching' (/home/circleci/project/venv/lib/python3.10/site-packages/sphinx/util/matching.py))
make: *** [Makefile:30: docs-build] Error 2
```

See, for example, https://app.circleci.com/pipelines/github/napari/docs/365/workflows/73b5e766-7082-4c2f-9774-dffa61a364e6/jobs/355?invite=true#step-107-2326_16

I will try to take a look.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -53,6 +53,8 @@ jobs:
           command: |
             . venv/bin/activate
             python -m pip install -r requirements.txt
+          environment:
+            PIP_CONSTRAINTS: ../resources/constraints/constraints_py3.10_docs.txt
       - run:
           name: Build docs
           command: |

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -52,9 +52,7 @@ jobs:
           name: Install python dependencies
           command: |
             . venv/bin/activate
-            python -m pip install -r requirements.txt
-          environment:
-            PIP_CONSTRAINTS: ../resources/constraints/constraints_py3.10_docs.txt
+            python -m pip install -r requirements.txt -c "napari/resources/constraints/constraints_py3.10_docs.txt"
       - run:
           name: Build docs
           command: |

--- a/.github/workflows/build_docs.yml
+++ b/.github/workflows/build_docs.yml
@@ -38,6 +38,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install "napari/[all]" -c "napari/resources/constraints/constraints_py3.10_docs.txt"
+          python -m pip install -r requirements.txt -c "napari/resources/constraints/constraints_py3.10_docs.txt"
 
       - name: Testing
         run: |

